src/ParticipationStorage/ParticipationStorageInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpab\Phpab\Storage;
+
+/**
+ * Stores the participation state of the user
+ */
+interface ParticipationStorageInterface
```

Comment:
Hmm I don't know where my other comment went but I said: "Shouldn't that be part of an Analytics namespace?"